src/Migrations/Version20190827081720.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Copyright 2019 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ *
+ * @SuppressWarnings(PHPMD)
+ */
+final class Version20190827081720 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Add fmt field to public key credential sources';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE users CHANGE icon icon VARCHAR(255) DEFAULT NULL');
```

Comment:
Are the alterations to the `users` schema intentional? If not, consider removing them from this migration. If these changes are a leftover from a previous Entity change, then consider opening a separate PR for fixing that. Or at least move it to a separate commit.